packages/react/src/component-tests/IcPopoverMenu/IcPopoverMenuData.tsx

```diff
@@ -0,0 +1,288 @@
+        <IcPopoverMenu submenuId="actions" aria-label="popover">
+          <IcMenuItem label="Edit" />
+          <IcMenuItem label="Find" />
+          <IcMenuItem label="Delete" variant="destructive" />
+        </IcPopoverMenu>
+      </>
+    </div>
+  );
+};
+
+export const DisabledPopoverMenu = () => {
+  const buttonEl = useRef<any>();
+  const popoverEl = useRef<any>();
+  const handleClick = () => {
+    popoverEl.current.open = buttonEl.current.dropdownExpanded;
+  };
+  const handleClose = () => {
+    buttonEl.current.dropdownExpanded = popoverEl.current.open;
+  };
+  return (
+    <div style={{ padding: "1rem" }}>
+      <>
+        <IcButton
+          id="popover-button"
+          dropdown
+          variant="primary"
+          onClick={() => handleClick()}
+          ref={buttonEl}
+        >
+          Button
+        </IcButton>
+        <IcPopoverMenu
+          anchor="popover-button"
+          aria-label="popover"
+          onIcPopoverClosed={() => handleClose()}
+          ref={popoverEl}
+        >
+          <IcMenuItem disabled={true} label="Copy code"></IcMenuItem>
+          <IcMenuItem disabled={true} label="Paste code"></IcMenuItem>
+          <IcMenuItem
+            disabled={true}
+            label="Actions"
+            submenuTriggerFor="actions"
+          />
+        </IcPopoverMenu>
+        <IcPopoverMenu submenuId="actions" aria-label="popover">
+          <IcMenuItem label="Edit" />
+          <IcMenuItem label="Find" />
+          <IcMenuItem label="Delete" variant="destructive" />
+        </IcPopoverMenu>
+      </>
+    </div>
+  );
+};
+
+export const DisableButton = () => {
+  const buttonEl = useRef<any>();
+  const popoverEl = useRef<any>();
+  const handleClick = () => {
+    popoverEl.current.open = buttonEl.current.dropdownExpanded;
+  };
+  const handleClose = () => {
+    buttonEl.current.dropdownExpanded = popoverEl.current.open;
+  };
+  return (
+    <div style={{ padding: "1rem" }}>
+      <>
+        <IcButton
+          disabled={true}
+          id="popover-button"
+          dropdown
+          variant="primary"
+          onClick={() => handleClick()}
+          ref={buttonEl}
+        >
+          Button
+        </IcButton>
+        <IcPopoverMenu
+          anchor="popover-button"
+          aria-label="popover"
+          onIcPopoverClosed={() => handleClose()}
+          ref={popoverEl}
+        >
+          <IcMenuItem disabled={true} label="Copy code"></IcMenuItem>
+          <IcMenuItem disabled={true} label="Paste code"></IcMenuItem>
+          <IcMenuItem
+            disabled={true}
+            label="Actions"
+            submenuTriggerFor="actions"
+          />
+        </IcPopoverMenu>
+        <IcPopoverMenu submenuId="actions" aria-label="popover">
+          <IcMenuItem label="Edit" />
+          <IcMenuItem label="Find" />
+          <IcMenuItem label="Delete" variant="destructive" />
+        </IcPopoverMenu>
+      </>
+    </div>
+  );
+};
```

Comment:
I think this example can be deleted as it's not being used in the test file anymore